docs/markdown/markseries.md

```diff
@@ -31,14 +31,11 @@ const myData = [
 ]
 ```
 
-react-vis offers two different types of MarkSeries, one that renders SVG and one that renders WebGL.
-The SVG mode is accessed by using the normal `MarkSeries`, just as above, while the WebGL mode is used by simply calling
-`MarkSeriesGL` instead of `MarkSeries`. Important tip! It is most effective to hover over your points using onNearestXY rather
-than onValueMouseOver.
+react-vis offers two different types of MarkSeries, one that renders SVG and one that renders Canvas.
+The SVG mode is accessed by using the normal `MarkSeries`, just as above, while the Canvas mode is used by simply calling `MarkSeriesCanvas` instead of `MarkSeries`.
 
 <!-- INJECT:"ScatterplotGLChart" -->
```

Comment:
I don't mean to keep you in endless revisions here, but ScatterplotGLChart should probably not be called that? It should ScatterplotCanvas or something? Same story with LineChartGl => LineChartCanvas.